nvm.sh

```diff
@@ -337,10 +337,10 @@ nvm_num_version_groups() {
     echo "0"
     return
   fi
-  VERSION="${VERSION#v*}"
-  VERSION="${VERSION%\.}"
+  VERSION="${VERSION#v}"
+  VERSION="${VERSION%.}"
   local NVM_NUM_DOTS
-  NVM_NUM_DOTS=$(echo "$VERSION" | command sed -e 's/[^\.]//g')
+  NVM_NUM_DOTS="${VERSION//[^.]/}"
```

Comment:
I'm afraid this form of substitution is not an option as it doesn't work in `dash`/`sh` - this will fail tests.